src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -55,6 +55,11 @@ export class CalciteActionBar {
     this.calciteActionBarToggle.emit();
   }
 
+  /**
+   * Used to set the a tooltip for the expand toggle.
```

Comment:
Nitpick: ` set the a tooltip` -> ` set a tooltip`

Reply:
🤦‍♂